openupgrade_scripts/scripts/hr_expense/15.0.2.0/post-migration.py

```diff
@@ -14,36 +14,36 @@ def _fill_payment_state(env):
     openupgrade.logged_query(
         env.cr,
         """
-        UPDATE account_move_line
-        SET exclude_from_invoice_tab=(coalesce(quantity, 0) = 0)
-        WHERE expense_id IS NOT NULL
+        UPDATE account_move_line AS aml
+        SET exclude_from_invoice_tab = true
+        FROM
+            account_account AS aa,
+            account_account_type AS aat
+        WHERE
+            aml.account_id = aa.id AND
+            aa.user_type_id = aat.id AND
+            aml.expense_id IS NOT NULL AND
+            aat.type = 'payable'
```

Comment:
```suggestion
        FROM
            account_account AS aa
        JOIN account_account_type AS aat
            ON aa.user_type_id = aat.id
        WHERE
            aml.account_id = aa.id AND
            aml.expense_id IS NOT NULL AND
            aat.type = 'payable'
```
it's nicer with JOIN

Reply:
@MiquelRForgeFlow thanks for the review! i didn’t know we could use joins there. i added your suggestion, but used `inner join` instead of just `join` (even though it means the same) because i prefer explicit over implicit.